demo_analyses/plot_scanpath.py

```diff
@@ -3,15 +3,27 @@
 Created on Mon Sep  5 11:06:43 2022
 
 @author: Marcus
+@contributer: Bob
 """
 
 import pandas as pd
 from pathlib import Path
-import matplotlib.pyplot as plt
 import os
-from psychopy import visual, core
+import cv2
+import time
+start = time.time()
 
 
+# %%
+
+def add_transparancy_cv2(cv2_func, image, alpha):
```

Comment:
this signature is funky. cv2_func is a copy of the image with lines added to it, and image is without lines, right? cv2_func  seems like the wrong name for that variable, threw me off

Reply:
I tried to make this code universal in one line for all drawing functions: 

https://gist.github.com/IAmSuyogJadhav/305bfd9a0605a4c096383408bee7fd5c

Overlay might indeed be a better variable name. Conceptually I was thinking I am gonna send a drawing function in some kind of wrapper function. But in the end I am just sending the overlay.

`image = add_transparancy_cv2(cv2.line(image.copy(), line_start, line_end, lineColor, thickness), image, alpha)`

could indeed just as easy be:

`image = cv2.addWeighted(cv2.line(image.copy(), line_start, line_end, lineColor, thickness), alpha, image, 1 - alpha, 0)`